src/stickybits.js

```diff
@@ -350,7 +356,8 @@ class Stickybits {
     // Only apply new styles if the state has changed
     if (state === it.state && stateChange === it.stateChange) return
     rAF(() => {
-      const stateStyles = {
+      const stateStyles = { };
+      stateStyles.element = {
```

Comment:
This is a breaking change. 

It looks like the updated parent class `[stickyParent]` could be added below line 361 within the classes object (also for adding the other parent css classes). 

This change will simplify the rest of the PR.

Reply:
@yowainwright Agree! I'll do the change a resubmit a review. Thanks

Reply:
Thank you!!! 🙏